app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -215,7 +222,8 @@ class CatalogueController @Inject() (
           envDatas                    = optLatestData.fold(envDatas)(envDatas + _),
           linkToLeakDetection         = urlIfLeaksFound,
           productionEnvironmentRoute  = serviceUrl,
-          serviceRoutes               = serviceRoutes
+          serviceRoutes               = serviceRoutes,
+          authorisation               = hasAuth
```

Comment:
both `authorisation` and `hasAuth` are vague in what auth we're talking about without looking at the type, and we could add more - wonder if they both should be called `hasBranchProtectionAuth`